src/main/java/org/terasology/Sample/blocks/RomanColumnFamily.java

```diff
@@ -50,15 +88,112 @@ private Block getProperBlock(WorldProvider worldProvider, Vector3i location) {
+            return BLOCKS.get(SideBitFlag.getSides(Side.BOTTOM, Side.TOP)); // Returns a normal column block
         }
         return null; // If we return null, something has gone terribly wrong
     }
 
+    private Block makeABlock(BlockFamilyDefinition definition, BlockBuilderHelper blockBuilder, String section, BlockUri blockUri, byte sides) {
+        Block newBlock = blockBuilder.constructSimpleBlock(definition, section);
+        newBlock.setUri(new BlockUri(blockUri, new Name(String.valueOf(sides))));
+        newBlock.setBlockFamily(this);
+
+        return newBlock;
+    }
+
+    @ReceiveEvent
+    public void onPlaceBlock(OnBlockItemPlaced event, EntityRef entity) {
+        BlockComponent blockComponent = event.getPlacedBlock().getComponent(BlockComponent.class);
+        if (blockComponent == null) {
+            return;
+        }
+
+        Vector3i targetBlock = blockComponent.getPosition();
+        processUpdateForBlockLocation(targetBlock);
+    }
+
+    private void processUpdateForBlockLocation(Vector3i blockLocation) {
+        for (Side side : new Side[] {Side.TOP, Side.BOTTOM}) {
+            Vector3i neighborLocation = new Vector3i(blockLocation);
+            neighborLocation.add(side.getVector3i());
+            if (worldProvider.isBlockRelevant(neighborLocation)) {
+                Block neighborBlock = worldProvider.getBlock(neighborLocation);
+                final BlockFamily blockFamily = neighborBlock.getBlockFamily();
+                if (blockFamily instanceof RomanColumnFamily) {
+                    RomanColumnFamily neighboursFamily = (RomanColumnFamily) blockFamily;
+                    Block neighborBlockAfterUpdate = neighboursFamily.getBlockForNeighborUpdate(neighborLocation, neighborBlock);
+                    if (neighborBlock != neighborBlockAfterUpdate) {
+                        worldProvider.setBlock(neighborLocation, neighborBlockAfterUpdate);
+                    }
+                }
+            }
+        }
+    }
+
+    @Override
+    public BlockUri getURI() {
+        return blockUri;
+    }
+
+    @Override
+    public String getDisplayName() {
+        return "Roman Column";
+    }
+
+    @Override
+    public Block getBlockForPlacement(Vector3i location, Side attachmentSide, Side direction) {
+        return getProperBlock(worldProvider, location);
+    }
+
+    @Override
+    public Block getArchetypeBlock() {
+        return BLOCKS.get((byte) 0);
+    }
+
+    @Override
+    public Block getBlockFor(BlockUri blockUri) {
+
+        for (Block block : BLOCKS.valueCollection()) {
+            if (block.getURI().equals(blockUri)) {
+                return block;
+            }
+        }
+        return null;
+
+    }
+
+    @Override
+    public Iterable<Block> getBlocks() {
+        return BLOCKS.valueCollection();
+    }
+
+    @Override
+    public Iterable<String> getCategories() {
+        return CATEGORIES;
```

Comment:
you actually don't need this and it doesn't do anything in terms of this block. if you look it up in the engine it acts as a multiplier based on the type of block.  so an axe is more efficient vs a tree then a shove. This is when this category will come into play.
https://github.com/MovingBlocks/Terasology/blob/develop/modules/Core/src/main/java/org/terasology/logic/health/BlockDamageAuthoritySystem.java#L206

if you look in the AbstractBlockFamily the functionality is already there. all you need is to set the categories in the constructor.